client/web/src/user/settings/repositories/RepositoryNode.tsx

```diff
@@ -190,6 +190,7 @@ export const CheckboxRepositoryNode: React.FunctionComponent<React.PropsWithChil
                         aria-label={`select ${name} repository`}
                         onChange={onClick}
                         checked={checked}
+                        label={<small></small>}
```

Comment:
```suggestion
                        label={<>
                            <StatusIcon mirrorInfo={mirrorInfo} />
                            <CodeHostIcon hostType={serviceType} />
                            <RepoLink
                                className="text-muted"
                                repoClassName="text-body"
                                repoName={name}
                                to={null}
                                onClick={handleOnClick}
                            />
                </>}
```

Is there a reason to not actually pass the elements we're using as the label here? @umpox or @valerybugakov do you have an insight? On a quick inspection, it seems like that could help align things as intended.

Reply:
I was considering a solution like this, I just wasn't sure if there was a reason that it wasn't in the label to begin with.

Reply:
Definitely worth asking (we'll see what Tom/Valery have to say). Especially if there isn't a comment explaining why the common pattern of using `label` would be skipped, it could have just been a mistake from moving quickly, doing a refactor, etc. And if there is a good reason it's like this, it's be a good opportunity to document it with a comment! :slightly_smiling_face:

Reply:
@umpox Follow up question: is there a reason to not make `label` required for the Checkbox component? Trying to do a fancy search to see if there are any other places there is no `label` for a Checkbox (per @jasongornall's first comment) but I haven't spent enough time to be successful. Just looking at the references might be quicker at this point 😆

Reply:
All the [Checkboxes](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24++%3CCheckbox.../%3E&patternType=structural) I just kind of scrolled through and looked for missing label= (There are a couple!)

Reply:
ha yeah.. I feel this would be useful https://sourcegraph.slack.com/archives/C03CSAER9LK/p1653670058462919?thread_ts=1653643033.212279&cid=C03CSAER9LK